File: ABC/ABC018/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int taro = sc.nextInt();
    int jiro = sc.nextInt();
    int saburo = sc.nextInt();
    sc.close();

    int t = 0;
    int j = 0;
    int s = 0;

    if (taro < jiro ) {
      t++;
    }

    if (taro < saburo) {
      t++;
    }

    if (jiro < saburo) {
      j++;
    }

    if (jiro < taro) {
      j++;
    }

    if (saburo < taro) {
      s++;
    }

    if (saburo < jiro ) {
      s++;
    }

    System.out.println(t + 1);
    System.out.println(j + 1);
    System.out.println(s + 1);
    
  }

}import java.util.*;
public class Main {
  public static void main(String[] args){
    Scanner sc = new Scanner(System.in);
    int s1 = sc.nextInt();
    int e1 = sc.nextInt();
    int s2 = sc.nextInt();
    int e2 = sc.nextInt();
    int s3 = sc.nextInt();
    int e3 = sc.nextInt();
    int sum = 0;
    sum = s1 * e1 / 10 + sum;
    sum = s2 * e2 / 10 + sum;
    sum = s3 * e3 / 10 + sum;
    System.out.println(sum);
  }
}

File: ABC/ABC018/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int taro, jiro, saburo;
  cin >> taro >> jiro >> saburo;

  if (taro < jiro) {
    if (taro < saburo) {
      cout << 3 << endl;
    } else {
      cout << 2 << endl;
    }
  } else {
    if (taro < saburo) {
      cout << 2 << endl;
    } else {
      cout << 1 << endl;
    }
  }

  if (jiro < saburo) {
    if (jiro < taro) {
      cout << 3 << endl;
    } else {
      cout << 2 << endl;
    }
  } else {
    if (jiro < taro) {
      cout << 2 << endl;
    } else {
      cout << 1 << endl;
    }
  }

  if (saburo < taro) {
    if (saburo < jiro) {
      cout << 3 << endl;
    } else {
      cout << 2 << endl;
    }
  } else {
    if (saburo < jiro) {
      cout << 2 << endl;
    } else {
      cout << 1 << endl;
    }
  }

}